.gitignore

```diff
@@ -17,3 +17,5 @@
 
 # Ignore Byebug command history file.
 .byebug_history
+
+.DS_Store
```

Comment:
i recommend adding this to a global gitignore on your machine instead.

https://help.github.com/articles/ignoring-files/